flashinfer/gemm/gemm_base.py

```
@@ -4002,17 +4005,78 @@ def mm_fp8(
     return out
 
 
+def _probe_mxfp8_gemm_tactics(module) -> List[int]:
+    """Probe which CUTLASS MXFP8 GEMM tactics can initialize on the current device.
+
+    SM120 (B100/B200 data-centre, ~256 KB shared memory/SM) and SM121 (GB10 /
+    DGX Spark GeForce Blackwell, ~99 KB shared memory/SM opt-in) share the same
+    compiled module.  Large CTA tiles — 256×128 and 128×256 — are compiled with
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)